packages/database/src/migrations/20201104032225-AddDashboardGroupForMS1Administration-modifies-data.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardReportId = 'Raw_Data_KI_MS1_Surveys';
+const dashboardGroupId = 331;
+const mapOverlayId = 'MS1_COMPLETENESS';
+exports.up = async function (db) {
+  // Add MS1 Administration to Dashboard Group
+  await db.runSql(`
+      INSERT INTO "dashboardGroup" (
+        "id",
+        "organisationLevel" ,
+        "userGroup" ,
+        "organisationUnitCode" ,
+        "dashboardReports" ,
+        "name",
+        "code",
+        "projectCodes")
+        
+      VALUES (
+         ${dashboardGroupId},
```

Comment:
I don't think you need to hardcode dashboardGroupId when inserting a new dashboard group. As I remember it is automatically generated. I think it's actually safer to let it self generated because other devs might insert a new dashboard group with the same id at the same time

Reply:
That's something that I didn't know. Thanks @chris-bes for sharing!